src/course-unit/breadcrumbs/Breadcrumbs.test.jsx

```diff
@@ -69,6 +70,39 @@ describe('<Breadcrumbs />', () => {
     });
   });
 
+  it('render Breadcrumbs with many ancestors items correctly', async () => {
+    axiosMock
+      .onGet(getCourseSectionVerticalApiUrl(courseId))
+      .reply(200, {
+        ...courseSectionVerticalMock,
+        ancestor_xblocks: [
+          {
+            children: [
+              {
+                url: '/course/course-v1:edX+DemoX+Demo_Course?show=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40sequential%2Bblock%4019a30717eff543078a5d94ae9d6c18a5',
```

Comment:
[question/optional]: Should we get this value from the data we use for mocks?

Reply:
Replaced